flaml/nlp/README.md

```diff
@@ -0,0 +1,9 @@
+How to use AutoTransformers:
+
```

Comment:
Please clarify the scope of use cases

Reply:
the current use cases that are supported 
the use cases that can be supported in future
the use cases that can't be supported in future (out of scope)